flow/output.go

```diff
@@ -67,3 +77,29 @@ func (oo Outputs) Init(ctx session.Context) {
 		go o.Init(ctx)
 	}
 }
+
+// Close closes all outputs.
+func (oo Outputs) Close(ctx session.Context) {
```

Comment:
This name seems a bit misleading; maybe `Wait` would be more appropriate?

I was expecting this to do something like `close(o.Done())` or cancel the context but it does neither.

Reply:
Or is this to satisfy the `io.ReadCloser` interface?

Reply:
No, it doesn't satisfy an interface, its just a bad name for the function.

Reply:
@bradbeam: https://github.com/rename-this/vhs/pull/10/commits/c3107ff9ab331b82d7743a0d107744d153e574aa